vnext/Microsoft.ReactNative.Cxx.UnitTests/JsonReader.cpp

```diff
@@ -0,0 +1,379 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Terms used in here:
+//   Container: represents a [] or {}
+//   isStringContent: is the current value a string? Either a name or quoted value
+
+#include "pch.h"
+#include "JsonReader.h"
+#include "Crash.h"
+
+namespace winrt::Microsoft::ReactNative::Bridge {
+
+static bool IsSimpleWhitespace(wchar_t c) noexcept {
```

Comment:
>IsSimpleWhitespace [](start = 12, length = 18)

This seems really similar to std::isspace

Reply:
It maybe similar, but not the same. We must follow here the JSON specification. It states that it must be only these four characters: [https://www.json.org/json-en.html](https://www.json.org/json-en.html)

---
In reply to: [356922292](https://github.com/microsoft/react-native-windows/pull/3760#discussion_r356922292) [](ancestors = 356922292)